development-docs/TESTING.md

```diff
@@ -148,29 +148,36 @@ The following table shows currently used tags:
+| travis             | Marks tests executed on Travis                                                     |
 | flaky              | Execute all flaky tests (tests, which are failing from time to time)               |
+| scalability        | Execute scalability tests                                                          |
+| specific           | Specific tests, which cannot be easily added to other categories                   |
 | nodeport           | Execute tests which use external lister of type nodeport                           |
 | loadbalancer       | Execute tests which use external lister of type loadbalancer                       |
-| bridge             | Execute tests which use Kafka Bridge                                               |
-| specific           | Specific tests, which cannot be easily added to other categories                   |
 | networkpolicies    | Execute tests which use Kafka with Network Policies                                |
-| tracing            | Execute tests for Tracing                                                          |
 | prometheus         | Execute tests for Kafka with Prometheus                                            |
-| oauth              | Execute tests which use OAuth                                                      |
+| tracing            | Execute tests for Tracing                                                          |
 | helm               | Execute tests which use Helm for deploy cluster operator                           |
-| olm                | Execute tests which use OLM for deploy cluster operator                            |
+| oauth              | Execute tests which use OAuth                                                      |
+| recovery           | Execute recovery tests                                                             |
 | connectoroperator  | Execute tests which deploy KafkaConnector resource                                 |
 | connect            | Execute tests which deploy KafkaConnect resource                                   |
 | connects2i         | Execute tests which deploy KafkaConnectS2I resource                                |
 | mirrormaker        | Execute tests which deploy KafkaMirrorMaker resource                               |
 | mirrormaker2       | Execute tests which deploy KafkaMirrorMaker2 resource                              |
 | conneccomponents   | Execute tests which deploy KafkaConnect, KafkaConnectS2I, KafkaMirrorMaker2, KafkaConnector resources |
+| bridge             | Execute tests which use Kafka Bridge                                               |
 | internalclients    | Execute tests which use internal (from pod) kafka clients in tests                 |
 | externalclients    | Execute tests which use external (from code) kafka clients in tests                |
+| olm                | Execute tests which use OLM for deploy cluster operator                            |
```

Comment:
This is actually not true. We can execute mostly all of the tests with CO installed via OLM. This tag should be for tests, which tests examples from Strimzi manifests.

Reply:
Thanks for correction :)